packages/web-react/src/components/Navigation/__tests__/useNavigationActionProps.test.ts

```diff
@@ -77,17 +77,17 @@ describe('useNavigationLinkProps', () => {
   });
 
   it('should return for elementType', () => {
-    const props: SpiritNavigationLinkProps<'div'> = {
+    const props: SpiritNavigationActionProps<'div'> = {
       elementType: 'div',
```

Comment:
Off topic: I'd prefer testing the real-world `<button>` type here.

Reply:
ok, I will add it when I introduce button styles to navigationaction